### PR TITLE
add beta exception for binding components

### PR DESCRIPTION
**Pull request by cicoyle**
**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)

## Description

Updating the conformance test requirement for `Beta` in the [certification lifecycle of components](https://docs.dapr.io/operations/components/certification-lifecycle/), to allow for the exception of binding components specifically to skip the `Beta` stage and go directly to `Stable` if the `The component is a binding and the certification tests are comprehensive`.
Pasted Graphic 2

## Issue reference

https://github.com/dapr/proposals/issues/37

### CODE REVIEW

**Comment by cicoyle** [comment]
I'm open to input on placement & wording if anyone has other ideas in mind

**Review by artursouza — APPROVED** [review]

File: daprdocs/content/en/operations/components/certification-lifecycle.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ All components start at the Alpha stage.
 - The component contains a record of the conformance test result reviewed and approved by Dapr maintainers with specific components-contrib version
 - Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases
 
+A component may skip the Beta stage and conformance test requirement per the discretion of the Maintainer if:
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/components/certification-lifecycle.md:52]:
```suggestion
{{% alert title="Note" color="primary" %}}
A component may skip the Beta stage and conformance test requirement per the discretion of the Maintainer if:
- The component is a binding
- The certification tests are comprehensive
{{% /alert %}}
```

File: daprdocs/content/en/operations/components/certification-lifecycle.md [hunk 2]
```diff
@@ -48,6 +48,9 @@ All components start at the Alpha stage.
...
+- The component is a binding and the certification tests are comprehensive
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/components/certification-lifecycle.md]:
```suggestion

```

Removing bc I wrapped it in alert shortcode above ^

**Review by hhunter-ms — CHANGES_REQUESTED** [review]

**Comment by hhunter-ms** [comment]
It looks like this resolves an issue I opened today: #3728 
Thanks!

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**